src/main/resources/assets/armorplus/lang/es_ES.lang

```diff
@@ -67,6 +67,46 @@ item.TheUltimateChestplate.name=El Peto Supremo
 item.TheUltimateLeggings.name=Las Grebas Supremas
 item.TheUltimateBoots.name=Las Botas Supremas
 
+#Swords
+item.armorplus.coal_sword.name=Espada de Carbón
+item.armorplus.lapis_sword.name=Espada de Lapislázuli
+item.armorplus.redstone_sword.name=Espada de Redstone
+item.armorplus.emerald_sword.name=Espada Esmeralda
+item.armorplus.obsidian_sword.name=Espada de Obsidiana
+item.armorplus.lava_sword.name=Espada de Lava
+item.armorplus.super_star_sword.name=Espada de Estrella Estupenda
+item.armorplus.guardian_sword.name=Espada de Guardián
+item.armorplus.ender_dragon_sword.name=Espada del Dragon del End
+
+#BattleAxes
+item.armorplus.coal_battle_axe.name=Hacha de Batalla de Carbón
+item.armorplus.lapis_battle_axe.name=Hacha de Batalla de Lapislázuli
+item.armorplus.redstone_battle_axe.name=Hacha de Batalla de Redstone
+item.armorplus.emerald_battle_axe.name=Hacha de Batalla Esmeralda
+item.armorplus.obsidian_battle_axe.name=Hacha de Batalla de Obsidiana
+item.armorplus.lava_battle_axe.name=Hacha de Batalla de Lava
+item.armorplus.super_star_battle_axe.name=Hacha de Batalla de Estrella Estupenda
+item.armorplus.guardian_battle_axe.name=Hacha de Batalla del Guardián
+item.armorplus.ender_dragon_battle_axe.name=Hacha de Batalla del Dragon del End
+
+#Bows
+item.armorplus.coal_bow.name=Arco de Carbón
+item.armorplus.lapis_bow.name=Arco de Lapislázuli
+item.armorplus.redstone_bow.name=Arco de Redstone
+item.armorplus.emerald_bow.name=Arco de Esmeralda
```

Comment:
Shouldn't this be `Arco Esmeralda` or the other emerald things be `Cosa de Esmeralda`?